text/survey_analysis_report2.md

```diff
@@ -0,0 +1,191 @@
+Many mentioned that it would only reach full impact if the <whole lab> would be using it, emphasizing the advantage of such a system for 
+collaborative work.
+In addition, they mentioned the time saved by not having to create a template for themselves,
+but only use an existing standard.
+While they were quite unanimous about using a template for new projects, they were critical about making a transition for ongoing projects,
+as the cost of transition might be too high.
+More generally, they were questioning the cost over benefit ratio of changing their current organization system,
+especially as learning a new workflow can take time.
+
+
+In many cases, people mentioned that the **files are organized via experiments**, not at the project level.
+In particular, people tend to pool data and code in a single experiment folder.
+This is reminiscent of the two example structure given in the library carpentry course [@LibraryCarpentryLcfairresearch2019].
+
+
+On the other hand, some **files are organised outside of project folders**, i.e.  in particular places irrespective of the project they belong to. 
+Many researchers reported having a folder for all conferences reports or all manuscritps, for instance.
+
+
+# The new template
+
+Data management principles recommend to keep all files related to a project in a single folder (it facilitates sharing of these files with the whole team working on the project), 
+and to manage data and code differently (it allows different version control system, as well as sharing and reusing data and code independently). 
+We designed a template that follows these principles, 
+but added some recommendations and technical solutions 
+in order to permit users to have cross-project and experiment-level organisations of their files.
+
+## The experiment level
+
+
+We propose to keep data and code in different first level folder, 
+and to create several new folders (in the different first level folders) for each new experiment. 
+This could be easily automated in your language of choice,
+and we are working on bringing this functionality to gin-tonic (see below). 
+In addition some of these new folders could also have their own templates. 
+For instance, you could use a specific BIDS template for some experiments, while others will use a different template.
+
+
+
+## The project and cross-project level
+
+We propose to mirror some folders (for shared figures, report and conference, and manuscripts) both in the project folder and in a different folder (fig. 4). 
+In addition, we would propose to add a `labcommons` folder that could be shared and synchronised between all lab members using cloud technologies. 
+This will be quite difficult to set manually (using alias folders for example), but using git submodule technology in gin tonic, it can be automated (see below).
```

Comment:
This all will be quite difficult to set up manually (using alias folders for example), but using the git submodule technology in GIN-tonic, it can be automated (see below).